### ISSUE TITLE
git repo conversion oddity/corruption

### ISSUE THREAD

**Issue by illume** [pygame/pygame#397]
tldr; there is a problem with the git conversion some time in the past.

I noticed this file is missing git history: https://github.com/pygame/pygame/blob/master/test/test_utils/arrinter.py See here history of that file here on bitbucket: https://bitbucket.org/pygame/pygame/history-node/48e19c7b9ee9/test/test_utils/arrinter.py?at=default

This commit from 1st Dec 2016 looks like a giant commit changing lots of files...
https://github.com/pygame/pygame/commit/eecbb2b2c1fdf27d3ba2ebc8e3df8ae39763e01c

Weird.

If you go back in time even further, the logs make no sense for this file at all...
```
git checkout 0da722280e031b0bc3f403dfa30db94d1a6d7733
git log test/test_utils/arrinter.py
```

- I've got a backup of the git repo from April 2017, which has the same issues. 
- We moved around 2017-03-26.
- Here's the email thread: "[The long-ish pygame github migration status email.](http://archives.seul.org/pygame/users/Mar-2017/msg00144.html)"

Based on that, I'm pretty sure this happened during the import somewhere. Either in the initial conversion to git, with the tool that links up peoples new github accounts with peoples old bitbucket/email accounts.

For new commits to github, I checked my email log, and it seems they are all making sense. But I haven't looked into this exhaustively.

Where to go from here? I'll do some more investigations about what is possible.

**Comment by illume** [comment]
Fixing this would require some deep git and hg hacking, and I don't see anyone finding the time to do this. The hg repo is still around if people want to search further back into the past.
